pkg/controller/authconfig_test.go

```diff
@@ -10,30 +10,40 @@ import (
 
 	"github.com/codeready-toolchain/registration-service/pkg/controller"
 	"github.com/codeready-toolchain/registration-service/pkg/configuration"
+	testutils "github.com/codeready-toolchain/registration-service/test"
 	"github.com/gin-gonic/gin"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
 )
 
-func TestAuthClientConfigHandler(t *testing.T) {
+type TestAuthConfigSuite struct {
+	testutils.UnitTestSuite
+}
+
+ func TestRunAuthClientConfigSuite(t *testing.T) {
+	suite.Run(t, &TestAuthConfigSuite{testutils.UnitTestSuite{}})
+}
+
+func (s *TestAuthConfigSuite) TestAuthClientConfigHandler() {
 	// Create a request to pass to our handler. We don't have any query parameters for now, so we'll
 	// pass 'nil' as the third parameter.
 	req, err := http.NewRequest(http.MethodGet, "/api/v1/authconfig", nil)
-	require.NoError(t, err)
+	require.NoError(s.T(), err)
 
 	// Create logger and registry.
 	logger := log.New(os.Stderr, "", 0)
 	configRegistry := configuration.CreateEmptyRegistry()
 
 	// Set the config for testing mode, the handler may use this.
 	configRegistry.GetViperInstance().Set("testingmode", true)
```

Comment:
You don't need to create a new CreatEmptyRegistry and set it to the testing mode now. Use `s.Config` where you need the configuration. It's already set to the testing mode for the suite. That was the main reason for introducing the suite: https://github.com/codeready-toolchain/registration-service/pull/29/files#diff-5626d2d6a4ce3579865081a105c86127R19-R27

Reply:
fixed here: 6f5b359318a99b71809fc28cc97837c6a906413b